include/openssl/conf.h

```diff
@@ -139,6 +139,10 @@ OPENSSL_EXPORT const char *NCONF_get_string(const CONF *conf,
 OPENSSL_EXPORT OPENSSL_DEPRECATED int CONF_modules_load_file(
     const char *filename, const char *appname, unsigned long flags);
 
+// CONF_get1_default_config_file returns a fixed dummy string. AWS-LC is defined
+// to have no config file options.
```

Comment:
Is returning null safer? Would customer code now accidentally attempt to open/read/write to a invalid file path and crash?

Reply:
hm, unguarded `strlen(path)` is unfortunate. that would segfault if `path` is `NULL`.

Reply:
Yeah I forgot to mention that this was the initial forcing function to return a string. Populating a value for the user to read into seemed like the better call than the potential segfault.